modules/administration-guide/attachments/restricted-environment/prepare-restricted-environment.sh

```diff
@@ -74,14 +77,27 @@ podman build -t "${my_operator_index}" -f "./${my_catalog}.Dockerfile" --no-cach
 oc patch OperatorHub cluster --type json \
     --patch '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]'
 
+# # Create OperatorGroup before deployment of catalog source
```

Comment:
is this section meant to be commeted out?

Reply:
It's to have something that works. Ideally a better jq filtering to keep only the latest bundle version, and minimze the number of images to  mirror.  Removing the bundle is not enough, we must also clean up the channel. Or the service refuses to start.

Using text based rather than SQLite based because SQLite based is advertised as on the way to deprecation.

Reply:
I apologize for erroneous reply yesterday. Comment display on a phone is not ideal.

This section was a leftover. AFAIK we don't need it.